<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookmanagerui</class>
 <widget class="QMainWindow" name="bookmanagerui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>484</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>7</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Rockwell</family>
    <pointsize>12</pointsize>
    <italic>true</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>bookmanagerui</string>
  </property>
  <widget class="QWidget" name="widget_main">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>484</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="welcome">
     <widget class="QLabel" name="lb_title1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>441</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>35</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>图书馆书籍管理系统</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lb_title2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>170</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>v1.0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_107">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>25</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>107studio出品</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_2regist">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>350</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Tai Le</family>
        <pointsize>15</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>注册账号</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_2login">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>350</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Tai Le</family>
        <pointsize>15</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>登陆账号</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_aboutus">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>350</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Tai Le</family>
        <pointsize>15</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>关于我们</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="regist">
     <widget class="QPushButton" name="pb_regist_back">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_regist">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>279</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>25</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>注册您的账号</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_username">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_account">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_password">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>290</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_isteacher">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>350</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>15</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>以老师身份注册（默认为学生身份）</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_regist">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>400</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>注册！</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
